sdk/datafactory/Azure.ResourceManager.DataFactory/CHANGELOG.md

```diff
@@ -1,16 +1,17 @@
 # Release History
 
-## 1.5.0-beta.1 (Unreleased)
+## 1.5.0 (2024-10-21)
```

Comment:
Need to specify a future date. Next time, please leave some buffer time for review/modification. Thanks.